src/generate-gallery.py

```diff
@@ -7,7 +7,7 @@
 from chemiscope.sphinx import ChemiscopeScraper
 
 
-HERE = os.path.realpath(os.path.dirname(__file__))
+HERE = os.path.realpath(os.path.join(os.path.dirname(__file__), "../"))
```

Comment:
```suggestion
ROOT = os.path.realpath(os.path.join(os.path.dirname(__file__), "../"))
```

Would be a better name

Reply:
hey I was just using the name that had been used ^^